mlflow/transformers/__init__.py

```diff
@@ -1309,7 +1309,7 @@ def _should_add_pyfunc_to_model(pipeline) -> bool:
         "DocumentQuestionAnsweringPipeline",
         "ImageToTextPipeline",
         "VisualQuestionAnsweringPipeline",
-        "ImageClassificationPipeline",
+        #"ImageClassificationPipeline",
```

Comment:
This can be deleted. No need to comment out.

Reply:
ok

Reply:
done